web/src/app/core/services/project.service.ts

```diff
@@ -147,4 +147,37 @@ export class ProjectService {
             { merge: true }))
       );
   }
+
+   // This function add the repository in any project
+   public saveMonitors(uid: string, monitors: MonitorModel[]): Observable<void> {
```

Comment:
I think this could go in its own service.

I probably need to move adding a repository to its own service too. I will create a ticket for the repository refactor

Reply:
sure will do it in that ticket

Reply:
created monitorService and added saveMonitors to it